### DISCUSSION TITLE
Extending Scratch synthetic voices' range

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7521] [Ideas] [answered]
I have no idea if this is even possible, but Scratch's synthetic voices are really... synthetic and flat (that is, bad). It would be cool if one could choose in a larger selection of voices, especially more natural voices with a natural intonation.

**Comment by DNin01** [comment, marked as answer]
We can't add new voices to the TTS extension without it not working for users who aren't using Scratch Addons, but we could try to make a way to insert AI voices as audio that can then be played with the "play sound" block.

If you're looking for TTS that can speak messages that aren't hard-coded into the project, such as user input, then https://github.com/TurboWarp/extensions might be a better place to develop that.
